mill-build/src/millbuild/Deps.scala

```diff
@@ -280,4 +280,15 @@ object Deps {
   object DocDeps {
     val millScip = mvn"io.chris-kipp::mill-scip_mill0.11:0.3.7"
   }
+
+  object AndroidDeps {
+    val manifestMergerVersion = "31.10.0"
+    val bundleToolVersion = "1.17.2"
+    val ndkVersion = "27.0.12077973"
+    val cmakeVersion = "3.22.1"
+    val layoutLibVersion = "15.1.2"
+    val composePreviewRendererVersion = "0.0.1-alpha09"
+    val uiToolingVersion = "1.7.6"
+  }
```

Comment:
What's missing here is to also use these dependencies somewhere, so we can auto-detect updates. Please add these to the `dummy` cross module in the top-level `build.mill`.

Reply:
cool, we'll do, what happens for the tools that we can't match to maven deps?

Reply:
Here's my draft:

* https://github.com/com-lihaoyi/mill/pull/5397


@vaslabs Don't know, if we need to accommodate to Android specifics. Feel free to comment or push to that PR branch.

Reply:
no worries, @souvlakias is on it, he's also referencing the dependencies where appropriate (e.g. for layout lib and compose renderer)

Reply:
If we need to customize the repos or other stuff, just create a second dummy module.